### DISCUSSION TITLE
Ionic UI compatibility

### DISCUSSION THREAD

**Discussion by aretrace** [wakujs/waku#810] [General]
Basic testing of [Ionic react components](https://ionicframework.com/docs/api) (i.e., importing relevant styles and a IonButton) does not currently work. First issue seems to be `@ionic/react/css/structure.css`.

**Comment by dai-shi** [comment]
Help wanted.

**Comment by jkhaui** [comment]
@aretrace ionic has never played nicely with react and SSR unfortunately - see, for example, [this discussion about remix and ionic](https://github.com/remix-run/remix/discussions/6253).

I suggest using a more modern alternative like [Konsta](https://github.com/konstaui/konsta) if you want iOS & material themed mobile components that are also SSR-friendly

**Reply by aretrace** [reply]
Yeah, I soon realized that after some initial tries. I've considered Konsta and others but if you want screen transitions you have to go with a framework with a custom router specialized in that (I could have seen if the View Transitions API could have mimic them but I did not want to go down that long path). What I was looking for did not have a requirement for SSR, but did require transition animations, so for now waku is simply not the right option (at least for now, [but things are getting closer](https://www.youtube.com/watch?v=eY6C_-aDdTo)).
